src/utils/classifiers.ts

```diff
@@ -48,6 +48,8 @@ export const sovAnalyticsCookie = { name: 'SovAnalytics', value: 'optout' };
 
 export const chartStorageKey = 'sovryn.charts';
 
+export const originsSaleStorageKey = 'sovryn.origins.sales';
```

Comment:
Remove this if local storage functionality is removed based on other suggested changes